libs/providers/flagd/README.md

```diff
@@ -42,20 +44,30 @@ Options can be defined in the constructor or as environment variables, with cons
   }))
 ```
 
-### Example Using a Unix Socket
+### Remote flag resolving using Unix Socket
 
 ```
   OpenFeature.setProvider(new FlagdProvider({
       socketPath: "/tmp/flagd.socks",
   }))
 ```
 
+### In-process resolver with gRPC sync connectivity
```

Comment:
Add a link [here](https://flagd.dev/concepts/syncs/#grpc-sync) so people understand what this is.

Reply:
I updated the readme with feedback. You can check the latest here - https://github.com/open-feature/js-sdk-contrib/blob/454e08bbdd1fd94ca5ed19b97050b24f3a691ec1/libs/providers/flagd/README.md 

Regarding the link, this usage is NOT flagd's sync compatibility on grpc. So I think the proposed link can mislead others. I don't think flagd docs have a dedicated section on sync contract that can be used by an in-process provider .

Reply:
What if we just linked to https://buf.build/open-feature/flagd/docs/main:flagd.sync.v1 ?

Reply:
I have a linked section in the latest readme. Will this be enough ? :) 

> Flag configurations for evaluation are obtained via gRPC protocol using [sync protobuf schema](https://buf.build/open-feature/flagd/file/main:sync/v1/sync_service.proto) service definition.`

Reply:
yep